bundle/deploy/files/delete.go

```diff
@@ -55,7 +38,7 @@ func (m *delete) Apply(ctx context.Context, b *bundle.Bundle) diag.Diagnostics {
 		return diag.FromErr(err)
 	}
 
-	cmdio.LogString(ctx, "Successfully deleted files!")
+	log.Debugf(ctx, "Successfully deleted files!")
```

Comment:
What's the value of keeping this around?

Reply:
removed